Project: Using Bootstrap

SmithVyne opened the pull request:
This project requests students to build a replica of the news site [Newsweek.com](https://www.newsweek.com/) using the Bootstrap framework.

You can find the original project specification at: [Odin Project](https://www.theodinproject.com/courses/html5-and-css3/lessons/using-bootstrap)

SmithVyne commented:
### This is a clone of the News Week homepage [here](https://www.newsweek.com/)

**Added Required Changes**
- I centered the element on the page
- I increased the font-size for the headers
- I validated my markup and corrected errors
- I improved the CSS file

I believe this project is complete and ready to be merged.
Thank you for your review

SofiBretz commented:
### Changes required ♻️
### Hello team 🌟

Great work on this project, but you need to fix the issues below before moving on ⬇️

- [ ] As the last TSE required: "In the CSS, always aim to apply DRY principles at all times." 
Make sure that basic CSS bad practices are not used (https://speckyboy.com/good-bad-css-practices/ for support ) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright) --> no repeating of the same rule. One example is the colors, do not repeat the same color all the time. Consider fixing this.

- [ ] As the last TSE required, Pull Request has a meaningful title and a short description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)) For example you commit: "Completed all major sections" what do you mean? try to be more descriptive. Consider making this in the upcoming commit when you arrange the CSS no repeating the same rule. 

_**After making the required changes please submit a new code review request**_
Good luck! 🍀 You are almost there! 💪

SofiBretz.

SmithVyne commented:
### Implemented DRY CSS principles and Updated Pull Request Message

- I joined all the hover pseudo-selectors in one location and merged selectors with similar style rules.

- I replaced COLOR: BLACK and COLOR: #f72210 with classes that I added to the appropriate elements in the markup.

- I replaced a frequent background color with a class.

- I updated the pull request title with a more descriptive text.
